frontend/src/components/Button.svelte

```diff
@@ -0,0 +1,23 @@
+<script lang="ts">
+    // your script goes here
```

Comment:
I don't think this comment brings any particular value.

Reply:
Nice catch. :)